src/lib/EventSubscriber/AppRendererSubscriber.php

```diff
@@ -51,7 +52,7 @@ public function renderApp(FilterResponseEvent $event)
 
         $response = $event->getResponse();
 
-        if ($response->isRedirect()) {
+        if ($response->isRedirect() || $response instanceof NotificationResponse) {
```

Comment:
unsure why you need to do that ? isn't it because https://jira.ez.no/browse/EZP-27582 / https://jira.ez.no/browse/EZP-27476 are not yet implemented ?